src/entities/models/project.ts

```diff
@@ -51,4 +51,15 @@ export class Project implements EditorEntityValue {
       identifier: this.identifier,
     };
   }
+
+  static newEmpty () : Project {
+    return new Project({
+      projectName: "new Project",
+      description: "project description",
```

Comment:
Personally I would capitalize the first letter, but thats more of a stylistic choice